### PR TITLE
 Migrate `DatesAndTimeline` component to Shadcn/Tailwind

### PR DESCRIPTION
**Pull request by 0xgun**
## Description
Migrate `DatesAndTimeline` component to Shadcn/Tailwind from Chakra

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/13946

### CODE REVIEW

**Comment by 0xgun** [comment]
@pettinarip could you please review this PR

File: src/components/Translatathon/DatesAndTimeline.tsx [hunk 1]
```diff
@@ -62,58 +61,49 @@ export const DatesAndTimeline = () => {
...
           <Flex
             key={index}
-            borderLeft={"1px solid"}
-            borderColor={
-              index === dates.length - 1 ? "transparent" : "primary.base"
-            }
-            px={4}
-            pb={index === dates.length - 1 ? 0 : 16}
-            gap={4}
+            className={`flex border-l ${index === dates.length - 1 ? "border-transparent" : "border-purple-600 dark:border-purple-400"} px-4 ${index === dates.length - 1 ? "pb-0" : "pb-16"} gap-4`}
```
Comment (pettinarip) [inline src/components/Translatathon/DatesAndTimeline.tsx:72]: Let's take advantage of the `cn` function for conditional styles. https://github.com/ethereum/ethereum-org-website/blob/dev/src/lib/utils/cn.ts

In this case we could have something like this
```tsx
className={cn("flex border-l", index === dates.length - 1 ? "border-transparent" : "border-primary", ....)}
```

Also, use `border-primary` for the purple color. This is a color from the DS and is already included. You can check them out in this line https://github.com/0xgun/ethereum-org-website/blob/dev/tailwind.config.ts#L164

File: src/components/Translatathon/DatesAndTimeline.tsx [hunk 2]
```diff
@@ -62,58 +61,49 @@ export const DatesAndTimeline = () => {
...
-                w={8}
-                h={8}
-                bg={isLive ? "primary.base" : "primary.lowContrast"}
-                borderRadius="full"
-                ml={-8}
+              <div
+                className={`-ml-8 h-8 w-8 rounded-full ${
+                  isLive
+                    ? "bg-purple-600 dark:bg-purple-400"
+                    : "bg-purple-100 dark:bg-purple-900"
```
Comment (pettinarip) [inline src/components/Translatathon/DatesAndTimeline.tsx:79]: `cn` funciton +
```suggestion
                  isLive
                    ? "bg-primary"
                    : "bg-primary-low-contrast"
```

File: src/components/Translatathon/DatesAndTimeline.tsx [hunk 3]
```diff
@@ -62,58 +61,49 @@ export const DatesAndTimeline = () => {
...
+              <Flex className="flex flex-col">
+                <h3 className="text-2xl">{date.title}</h3>
+                <p>{date.description}</p>
               </Flex>
               {date.link && (
                 <Flex>
                   <ButtonLink
                     href={date.link}
-                    mt={2}
+                    className={`mt-2 ${date.link === APPLICATION_URL && !isLive ? "pointer-events-none text-disabled" : ""}`}
```
Comment (pettinarip) [inline src/components/Translatathon/DatesAndTimeline.tsx:104]: Use `cn` function.

**Review by pettinarip — CHANGES_REQUESTED** [review]
@0xgun looks good visually but left a few comments to use the `cn` function and defined DS colors.

**Comment by 0xgun** [comment]
Hey @pettinarip , I have made all the requested changes

File: src/components/Translatathon/DatesAndTimeline.tsx [hunk 4]
```diff
@@ -62,58 +63,57 @@ export const DatesAndTimeline = () => {
...
               <Flex
-                h={8}
-                bg={isLive ? "primary.base" : "primary.lowContrast"}
-                borderRadius="full"
-                px={4}
-                alignItems="center"
-                color={isLive ? "background.base" : "body.base"}
+                className={`flex h-8 items-center rounded-full px-4 ${
+                  isLive
+                    ? "dark:text-ButtonLinkack bg-purple-600 text-white dark:bg-purple-400"
```
Comment (pettinarip) [inline src/components/Translatathon/DatesAndTimeline.tsx]:
```suggestion
                    ? "dark:text-back bg-purple-600 text-white dark:bg-purple-400"
```

File: src/components/Translatathon/DatesAndTimeline.tsx [hunk 5]
```diff
@@ -62,58 +63,57 @@ export const DatesAndTimeline = () => {
...
-                    mt={2}
+                    className={cn(
+                      "mt-2",
+                      date.link === APPLICATION_URL && !isLive
+                        ? "pointer-events-none text-disabled"
+                        : ""
+                    )}
                     variant="outline"
-                    isDisabled={date.link === APPLICATION_URL && !isLive} // Application only
+                    aria-disabled={date.link === APPLICATION_URL && !isLive}
```
Comment (pettinarip) [inline src/components/Translatathon/DatesAndTimeline.tsx:117]: 👍🏼

**Review by pettinarip — APPROVED** [review]
🚀

**Pull request merged**